.github/workflows/workflow.yml

```diff
@@ -60,5 +70,13 @@ jobs:
     - name: push docker image
       run: docker push craigbeasley/todo_app:prod
 
+    - name: set up terraform
+      run: terraform init
+    
+    - name: terraform apply
+      run: terraform apply -auto-approve
+
     - name: update Azure 
-      run: curl -dH --fail -X POST '${{secrets.AZURE_WEB_HOOK}}' 
+      run: curl -dH --fail -X POST "$(terraform output -raw webhook)"
```

Comment:
It looks like you haven't tried running this new deployment setup (the step is being skipped by CI). Have a go before completing the PR to check that everything is running as expected

Reply:
Apologies, I can see the deployment ran successfully in the past. Ignore this